content/posts/February-2022.md

```diff
@@ -25,3 +25,15 @@ weight: 64
   ([packit#1483](https://github.com/packit/packit/pull/1483))
   ([packit.dev#390](https://github.com/packit/packit.dev/pull/390))
 - `python3-setuptools_scm` is now available by default in the sandbox.
+-
+
+## Week 8 (February 18th - February 28th)
+
+- Every new source-git commit generated by a dist-git update has a new
+  _git trailer_. `From-dist-git-commit` is the trailer holding the
+  _hash_ of the dist-git commit triggering the source-git commit.
+- A new option, `srpm_build_deps`, in the packit configuration file  
+  can be used. Which is a list of RPM dependencies that are needed for
+  the actions to be run when building SRPM and is an indicator to build
+  the SRPMs in Copr.
+- The packit docker image is now configured to properly handle `fkinit`.
```

Comment:
This was mostly a bugfix, not a new or important feature. I would not include it.